applications/app/AppLoader.scala

```diff
@@ -43,7 +49,8 @@ trait ApplicationsServices {
   lazy val facebookGraphApiClient = wire[FacebookGraphApiClient]
   lazy val facebookGraphApi = wire[FacebookGraphApi]
   lazy val newsletterApi = wire[NewsletterApi]
-  lazy val emailEmbedAgent = wire[EmailEmbedAgent]
+  lazy val emailEmbedAgent = wire[EmailSignupAgent]
+  lazy val groupedNewslettersAgent = wire[GroupedNewslettersAgent]
```

Comment:
Maybe we should go with either `Newsletter` or `Email` consistently in class names?